drivers/dac/ltc2672/ltc2672.h

```diff
@@ -68,9 +68,10 @@
 #define LTC2672_MAX_CONFIG_MASK  		15
 #define LTC2672_MAX_TOGGLE_MASK  		31
 #define LTC2672_BIT_SHIFT_12BIT			4
-#define LTC2672_NUM_MUX_SELCETS			22
+#define LTC2672_NUM_MUX_SELECTS			22
 #define LTC2672_NUM_CURRENT_SPANS		10
 #define LTC2672_NUM_FAULTS				7
+#define MICROAMP_PER_MILLIAMP			1000
```

Comment:
this should go in `no_os_units.h` in a separate commit, prior to your driver patch.
https://github.com/analogdevicesinc/no-OS/blob/0cd0ac78cd95f13a14ae32a2daeb430c0444ba3b/include/no_os_units.h#L68-L70

Reply:
Shall I open a separate PR for this and then use the no_os_util for this PR?

Reply:
You can do a separate commit in this PR. Prior to the commits for ltc2672